public/app/features/teams/__snapshots__/TeamSettings.test.tsx.snap

```diff
@@ -16,14 +16,14 @@ exports[`Render should render component 1`] = `
           "permission": 0,
         }
       }
-      disabled={false}
+      disabled={true}
```

Comment:
has this changed because we're now correctly importing the `contextSrv`? 🤔

Reply:
yeah, I think that's the reason but not sure why. In any case I'm gonna refactor TeamSettings next so this snapshot will go away.

Reply:
just making sure it's not started disabling something it shouldn't have 😂 something going from `false` -> `true` is always a bit suspicious